tests/mocks/MockFactory.ts

```diff
@@ -0,0 +1,10 @@
+import { anything, instance, mock, when } from 'ts-mockito';
+import StorageService from '../../services/StorageService';
+
+export default class Mocks {
+  public static storage(fileLocation: string): StorageService {
+    const storageMock = mock(StorageService);
+    when(storageMock.uploadToFolder(anything(), anything(), anything(), anything())).thenResolve(fileLocation);
+    return instance(storageMock);
+  }
+}
```

Comment:
We also need to mock the other StorageService functions here, so that no actual S3 calls get made. Right now, only `uploadToFolder` is the only fn that's mocked, but we also need to mock `upload` and `delete`.